### ISSUE TITLE
unrolling issue with PlaceHolderSynthesizer

### ISSUE THREAD

**Issue by RobinDavid** [quarkslab/qsynthesis#1]
The ``PlaceHolderSynthesizer`` seem's not to properly replace placeholder variable during the synthesis process, on some use-cases. The resulting synthesized expression is thus completely wrong. I need to dig deeper.

**Comment by solitudecc** [comment]
Can it be repaired?

**Comment by RobinDavid** [comment]
Yes, definitely. I mention it here for the record, I will give it look when will have a minute.

**Comment by RobinDavid** [comment]
No. Not yet.

**Comment by solitudecc** [comment]
Hello, can you add your telegram?

**Comment by RobinDavid** [comment]
> Hello, can you add your telegram?

I don't have. You can DM me on Twitter if needed

**Comment by solitudecc** [comment]
thank you
